test/functional/services/saved_objects/utils.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { join } from 'path';
+// @ts-ignore
+import shell from 'shelljs';
+
+export const finalDirAndFile = (destDir: string) => (filePath = 'exported.ndjson') => {
+  const destFilePath = join(destDir, filePath);
```

Comment:
I'm not sure we need new code here (this utils.ts)?  We have other code that cleans and recreates directories for screenshots using mkdirAsync like here;
https://github.com/elastic/kibana/blob/master/test/functional/services/common/screenshots.ts#L54

Reply:
Are you saying we should use `mkDirAsync` instead?  I can refactor that real quick.

Reply:
I should point out, this fn does not create anything, just used to join strings.